src/platforms/flutter/usage/advanced-usage.mdx

```diff
@@ -16,8 +37,8 @@ For the usage of the Flutter SDK, the minimal required Dart SDK version is `2.8.
 
 ## Known limitations
 
-- Flutter `split-debug-info` flag isn't supported yet, if this feature is enabled, Dart stack traces are not useful
-- Flutter `obfuscate` flag isn't supported yet, if this feature is enabled, Dart stack traces are not useful
+- Flutter `split-debug-info` and `obfuscate` flag aren't supported on iOS yet, but only on Android, if this feature is enabled, Dart stack traces are not human readable
+- If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.
 
 ## Tips for catching errors
```

Comment:
Would you rather this content be on a Troubleshooting page? (similar to https://docs.sentry.io/platforms/javascript/troubleshooting/)

```suggestion
## Tips for Catching Errors
```